elixir/apps/domain/lib/domain/network.ex

```diff
@@ -2,20 +2,28 @@ defmodule Domain.Network do
   alias Domain.Repo
   alias Domain.Network.Address
 
-  @cidrs %{
-    # Notice: those are also part of "resources_account_id_cidr_address_index" DB constraint
+  # Encompasses all CIDR and DNS Resource addresses
+  @reserved_cidrs %{
```

Comment:
@jamilbk Yeah, I support being more restrictive. As I remember, this is the validation that we had before but was changed because we had a client that used CGNAT ranges so we did not exclude it completely.